uaclient/entitlements/base.py

```diff
@@ -46,6 +46,12 @@ def description(self) -> str:
         """A sentence describing this entitlement"""
         pass
 
+    @property
+    @abc.abstractmethod
+    def reboot_on_enable(self) -> bool:
```

Comment:
Could this just be `reboot_on_enable = False`?

Reply:
correct. good thought. dropped it and focused this changeset specifically on the FIPS subclasses untill we have a 2nd use-case that requires reboot post enable.